src/olympia/abuse/templates/abuse/appeal.html

```diff
@@ -29,6 +29,10 @@ <h3>{{ _('Decision {0}')|format_html(decision_id) }}</h3>
                 {% endif %}
                 {% if appeal_email_form or appeal_form %}
                     <p> <input id="appeal-submit" type="submit" value="{{ _('OK') }}"> </p>
+                {% elif appealed_decision_already_made %}
+                    <p> {{ _("Thank you for your report.") }} </p>
+                    <p> {{ _("We have already reviewed a similar appeal from another reporter, and have reversed our prior decision. We have taken action against the content and/or account holder in accordance with our policies.") }}</p>
```

Comment:
Is this always a true statement though?  Sometimes we've reviewed an appeal and decided to uphold the original decision to not take down the content.

Reply:
Grr, you're right. I specifically asked for an updated copy that could cover that but then copy/pasted the updated copy, forgetting that scenario.

Reply:
The doc has a separate copy for this so I'll update the PR.

Reply:
https://github.com/mozilla/addons-server/pull/21788/commits/c0c29e1c8568d0e6427c7f05eda8990d4ed2b6f1